pages/news/2020-XX-XX-concrete-action.html

```diff
@@ -0,0 +1,63 @@
+
+Although we eliminated the use of the word "slave" back in [2016](https://github.com/mixxxdj/mixxx/commit/e59916caf72a256bb28b1722759a629c5cc9cf81), "master" still appears in a number of places.
+We have a "master" code branch, a "Master Sync" feature, an "EngineMaster" object, and more.
+We are working to migrate our code away from these words, even in cases where the particular meaning of "master" was never explicitly associated with a problematic context.
+
+It’s important to note that there will be some cases where the word will remain.
+Many DJ controllers have buttons marked "master" on them, and we need to refer to those buttons by name.
+There are also the audio engineering concepts of a "master tape" or "audio mastering", so there may be areas of our documentation or code that would be less understandable if we tried to use synonyms.
+But in situations where master is used in code and previously was part of a master/slave terminology pairing, we commit to changing the language we use by DATEDATEDATE **TKTKTKTK commit to a timeline to change the main branch?? TKTKTKTKTK**.
+
+But changing language is not enough.
+
+## Diversifying Recruiting
+
+In our discussions, we quickly noted that the Mixxx contributor base is largely white, male, and either from the US or Western Europe.
+For some of our coders, the experience of working on Mixxx has been a doorway to employment at some of the most prestigious software companies, and we want that door to be open to anyone.
+It is our goal to make the Mixxx project as inclusive, accessible and welcoming as possible.
+As such, it is our responsibility to actively seek out and promote the participation of traditionally underserved groups.
+
+As a small volunteer-led project with no regular income, budget, or employees, we don't yet know for sure what the demographic makeup of our contributors is -- but we are aware of the research that shows that blind admission and hiring tends to result in less diverse outcomes.
+To that end we plan to participate in the [Outreachy](https://www.outreachy.org/) program and have a paid internship position by December 2020, the next available cycle.
+Outreachy’s stated goal is to provide "internships to work in open source and free software, for women (both cis and trans), trans men, and genderqueer people" and "residents and nationals of the United States of any gender who are Black/African American, Hispanic/Latin@, Native American/American Indian, Alaska Native, Native Hawaiian, or Pacific Islander."
+They encourage "anyone who faces under-representation, systemic bias, or discrimination in the technology industry of their country" to apply.
+
+Our goal is that this Outreachy intern will be the first of an increasing number of valued contributors from underserved communities that will help us be a healthier, more sustainable open source project.
```

Comment:
```suggestion
Our goal is that this Outreachy intern will be the first of an increasing number of valued contributors from more diverse backgrounds that will help us be a healthier, more sustainable open source project.
```
"Underserved" feels oddly euphemistic to me.

Reply:
it's a very common phrasing in social justice discourse

Reply:
we shouldn't expect that our users are familiar with the social justice discourse though :smile: Do you think the wording that @Be-ing is worse? If not, I think we should change it IMHO.

Reply:
I'm still uncomfortable with this language. I cannot recall seeing "underserved" used in social justice discourse. Where I have seen it used is in discourse of outsiders writing about "underserved communities", such as mainstream media.

Reply:
Oh, I see you changed this :+1:

Reply:
(for the record, "underserved" is a pretty common phrase: https://www.google.com/search?client=ubuntu&channel=fs&q=underserved+communities+black+lives+matter&ie=utf-8&oe=utf-8)